hedera-node/src/main/java/com/hedera/services/store/models/AccountConversion.java

```diff
@@ -39,12 +45,19 @@ public static void mapMerkleToModel(MerkleAccount merkle, Account model) {
 		model.setAutoRenewSecs(merkle.getAutoRenewSecs());
 		model.setDeleted(merkle.isDeleted());
 		model.setSmartContract(merkle.isSmartContract());
-	} 
-	
+	}
+
+	/**
+	 * Maps properties between a model {@link Account} and a {@link MerkleAccount}
+	 *
+	 * @param model - the Account model, which will be used to map into a MerkleAccount
+	 * @param merkle - the merkle account
+	 */
 	public static void mapModelToMerkle(Account model, MerkleAccount merkle) {
 		if (model.getProxy() != null) {
 			merkle.setProxy(model.getProxy().asEntityId());
 		}
+		merkle.setMemo(model.getMemo());
```

Comment:
I think that when implementing mapping methods we would benefit from a proper ordering of the mapped properties within the method, which follows the order of the properties in the model.